.github/workflows/presubmit.yaml

```diff
@@ -47,6 +47,13 @@ jobs:
       run: cd android-cuttlefish/frontend && debuild -i -us -uc -b
     - name: install user debian package
       run: dpkg -i android-cuttlefish/cuttlefish-user_*_*64.deb || apt-get install -f -y
+    - name: Create artifact
+      run: cd android-cuttlefish && apt-get install -y p7zip-full && 7z a -mx=9 cuttlefish_packages_x86.7z *.deb
+    - name: Publish cuttlefish_packages_x86.7z
+      uses: actions/upload-artifact@v3
```

Comment:
nit: should use a commit hash

Reply:
Done.

Reply:
Done. Change to v4.0.0 commit hash, since v3 will be deprecated soon.